rancher2/resource_rancher2_cluster.go

```diff
@@ -327,6 +338,12 @@ func clusterStateRefreshFunc(client *managementClient.Client, clusterID string)
 			return nil, "", err
 		}
 
+		// If we encounter an error, such as "only one of zone or region must be specified"
```

Comment:
Is this our code? It's not in upstream

Reply:
yes, it is https://github.com/confluentinc/terraform-provider-rancher2/pull/6

Reply:
Looks like I also binned this off after @thomas-maurice added it...
https://github.com/confluentinc/terraform-provider-rancher2/commit/aac66e479ccd958e937bf614056299367f9d7f0f

🤦‍♂